.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+	// See http://go.microsoft.com/fwlink/?LinkId=827846
+	// for the documentation about the extensions.json format
+	"recommendations": [
+		"platformio.platformio-ide"
+	]
+}
```

Comment:
remove the vscode settings from git, you can keep it local, but don't commit them

Reply:
I will remove it